requirements.txt

```diff
@@ -1,6 +1,7 @@
 beautifulsoup4
 requests
 requests[security]
+pathlib
```

Comment:
pathlib[1] comes standard with python >= 3.4, there is no need for this in the requirements.

[1] https://pypi.org/project/pathlib/